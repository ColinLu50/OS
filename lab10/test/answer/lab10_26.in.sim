23505
776
778
634
776
630
