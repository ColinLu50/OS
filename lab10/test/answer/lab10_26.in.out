FCFS
139 -> 263 -> 188 -> 580 -> 489 -> 259 -> 636 -> 639 -> 438 -> 372 -> 150 -> 147 -> 243 -> 179 -> 380 -> 102 -> 190 -> 401 -> 513 -> 98 -> 305 -> 355 -> 636 -> 352 -> 235 -> 443 -> 527 -> 392 -> 306 -> 499 -> 215 -> 488 -> 501 -> 92 -> 487 -> 208 -> 221 -> 126 -> 63 -> 135 -> 1 -> 292 -> 286 -> 295 -> 506 -> 181 -> 473 -> 361 -> 486 -> 71 -> 99 -> 480 -> 639 -> 62 -> 17 -> 136 -> 202 -> 100 -> 605 -> 611 -> 224 -> 12 -> 556 -> 477 -> 306 -> 92 -> 472 -> 198 -> 301 -> 621 -> 632 -> 603 -> 618 -> 537 -> 52 -> 107 -> 351 -> 401 -> 236 -> 115 -> 148 -> 188 -> 27 -> 450 -> 196 -> 8 -> 279 -> 422 -> 426 -> 17 -> 353 -> 547 -> 518 -> 332 -> 624 -> 393 -> 65 -> 634 -> 59 -> 561 -> 513 -> 561 -> 32 -> 327 -> 53 -> 206 -> 423 -> 437 -> 324 -> 578 -> 412 -> 533 -> 367 -> 503 -> 519 -> 155 -> 482 -> 195 -> 366
23505
SSTF
139 -> 136 -> 135 -> 126 -> 115 -> 107 -> 102 -> 100 -> 99 -> 98 -> 92 -> 92 -> 71 -> 65 -> 63 -> 62 -> 59 -> 53 -> 52 -> 32 -> 27 -> 17 -> 17 -> 12 -> 8 -> 1 -> 147 -> 148 -> 150 -> 155 -> 179 -> 181 -> 188 -> 188 -> 190 -> 195 -> 196 -> 198 -> 202 -> 206 -> 208 -> 215 -> 221 -> 224 -> 235 -> 236 -> 243 -> 259 -> 263 -> 279 -> 286 -> 292 -> 295 -> 301 -> 305 -> 306 -> 306 -> 324 -> 327 -> 332 -> 351 -> 352 -> 353 -> 355 -> 361 -> 366 -> 367 -> 372 -> 380 -> 392 -> 393 -> 401 -> 401 -> 412 -> 422 -> 423 -> 426 -> 437 -> 438 -> 443 -> 450 -> 472 -> 473 -> 477 -> 480 -> 482 -> 486 -> 487 -> 488 -> 489 -> 499 -> 501 -> 503 -> 506 -> 513 -> 513 -> 518 -> 519 -> 527 -> 533 -> 537 -> 547 -> 556 -> 561 -> 561 -> 578 -> 580 -> 603 -> 605 -> 611 -> 618 -> 621 -> 624 -> 632 -> 634 -> 636 -> 636 -> 639 -> 639
776
SCAN
139 -> 136 -> 135 -> 126 -> 115 -> 107 -> 102 -> 100 -> 99 -> 98 -> 92 -> 71 -> 65 -> 63 -> 62 -> 59 -> 53 -> 52 -> 32 -> 27 -> 17 -> 12 -> 8 -> 1 -> 0 -> 147 -> 148 -> 150 -> 155 -> 179 -> 181 -> 188 -> 190 -> 195 -> 196 -> 198 -> 202 -> 206 -> 208 -> 215 -> 221 -> 224 -> 235 -> 236 -> 243 -> 259 -> 263 -> 279 -> 286 -> 292 -> 295 -> 301 -> 305 -> 306 -> 324 -> 327 -> 332 -> 351 -> 352 -> 353 -> 355 -> 361 -> 366 -> 367 -> 372 -> 380 -> 392 -> 393 -> 401 -> 412 -> 422 -> 423 -> 426 -> 437 -> 438 -> 443 -> 450 -> 472 -> 473 -> 477 -> 480 -> 482 -> 486 -> 487 -> 488 -> 489 -> 499 -> 501 -> 503 -> 506 -> 513 -> 518 -> 519 -> 527 -> 533 -> 537 -> 547 -> 556 -> 561 -> 578 -> 580 -> 603 -> 605 -> 611 -> 618 -> 621 -> 624 -> 632 -> 634 -> 636 -> 639
778
C-SCAN
139 -> 136 -> 135 -> 126 -> 115 -> 107 -> 102 -> 100 -> 99 -> 98 -> 92 -> 71 -> 65 -> 63 -> 62 -> 59 -> 53 -> 52 -> 32 -> 27 -> 17 -> 12 -> 8 -> 1 -> 0 -> 642 -> 639 -> 636 -> 634 -> 632 -> 624 -> 621 -> 618 -> 611 -> 605 -> 603 -> 580 -> 578 -> 561 -> 556 -> 547 -> 537 -> 533 -> 527 -> 519 -> 518 -> 513 -> 506 -> 503 -> 501 -> 499 -> 489 -> 488 -> 487 -> 486 -> 482 -> 480 -> 477 -> 473 -> 472 -> 450 -> 443 -> 438 -> 437 -> 426 -> 423 -> 422 -> 412 -> 401 -> 393 -> 392 -> 380 -> 372 -> 367 -> 366 -> 361 -> 355 -> 353 -> 352 -> 351 -> 332 -> 327 -> 324 -> 306 -> 305 -> 301 -> 295 -> 292 -> 286 -> 279 -> 263 -> 259 -> 243 -> 236 -> 235 -> 224 -> 221 -> 215 -> 208 -> 206 -> 202 -> 198 -> 196 -> 195 -> 190 -> 188 -> 181 -> 179 -> 155 -> 150 -> 148 -> 147
634
LOOK
139 -> 136 -> 135 -> 126 -> 115 -> 107 -> 102 -> 100 -> 99 -> 98 -> 92 -> 71 -> 65 -> 63 -> 62 -> 59 -> 53 -> 52 -> 32 -> 27 -> 17 -> 12 -> 8 -> 1 -> 147 -> 148 -> 150 -> 155 -> 179 -> 181 -> 188 -> 190 -> 195 -> 196 -> 198 -> 202 -> 206 -> 208 -> 215 -> 221 -> 224 -> 235 -> 236 -> 243 -> 259 -> 263 -> 279 -> 286 -> 292 -> 295 -> 301 -> 305 -> 306 -> 324 -> 327 -> 332 -> 351 -> 352 -> 353 -> 355 -> 361 -> 366 -> 367 -> 372 -> 380 -> 392 -> 393 -> 401 -> 412 -> 422 -> 423 -> 426 -> 437 -> 438 -> 443 -> 450 -> 472 -> 473 -> 477 -> 480 -> 482 -> 486 -> 487 -> 488 -> 489 -> 499 -> 501 -> 503 -> 506 -> 513 -> 518 -> 519 -> 527 -> 533 -> 537 -> 547 -> 556 -> 561 -> 578 -> 580 -> 603 -> 605 -> 611 -> 618 -> 621 -> 624 -> 632 -> 634 -> 636 -> 639
776
C-LOOK
139 -> 136 -> 135 -> 126 -> 115 -> 107 -> 102 -> 100 -> 99 -> 98 -> 92 -> 71 -> 65 -> 63 -> 62 -> 59 -> 53 -> 52 -> 32 -> 27 -> 17 -> 12 -> 8 -> 1 -> 639 -> 636 -> 634 -> 632 -> 624 -> 621 -> 618 -> 611 -> 605 -> 603 -> 580 -> 578 -> 561 -> 556 -> 547 -> 537 -> 533 -> 527 -> 519 -> 518 -> 513 -> 506 -> 503 -> 501 -> 499 -> 489 -> 488 -> 487 -> 486 -> 482 -> 480 -> 477 -> 473 -> 472 -> 450 -> 443 -> 438 -> 437 -> 426 -> 423 -> 422 -> 412 -> 401 -> 393 -> 392 -> 380 -> 372 -> 367 -> 366 -> 361 -> 355 -> 353 -> 352 -> 351 -> 332 -> 327 -> 324 -> 306 -> 305 -> 301 -> 295 -> 292 -> 286 -> 279 -> 263 -> 259 -> 243 -> 236 -> 235 -> 224 -> 221 -> 215 -> 208 -> 206 -> 202 -> 198 -> 196 -> 195 -> 190 -> 188 -> 181 -> 179 -> 155 -> 150 -> 148 -> 147
630
